dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/WorkspaceItemConverter.java

```diff
@@ -86,6 +85,10 @@ public WorkspaceItemRest fromModel(org.dspace.content.WorkspaceItem obj) {
         // info
 
         if (collection != null) {
+            // we set the status to true as we will discover validation error later in this block
+            // we could eventually leave the status to empty if we don't have collection information, this could be
+            // eventually the case when projection support will be included
```

Comment:
IMHO it is not a TODO or FIXME right now... it is more a comment to remind my(our)self why it could be useful to have the status as a Boolean instead than a boolean. But if you think more appropriate to flag it as a TODO or FIXME I will make the change

Reply:
I was just worried that the comment might get lost/forgotten.  Flagging as TODO or FIXME ensures that (at least some) IDEs will highlight the note.  But, I don't feel strongly on this, so I'm fine with leaving this as-is, since you seem to have a plan to clean this up in the near future.